app.py

```diff
@@ -1,22 +1,61 @@
 from distutils.log import debug
-from flask import Flask
+from flask import Flask, request, render_template, redirect, flash, session
 from flask_debugtoolbar import DebugToolbarExtension
+import surveys
 
 app = Flask(__name__)
 app.config['SECRET_KEY'] = "password"
 debug = DebugToolbarExtension(app)
+app.config['DEBUG_TB_INTERCEPT_REDIRECTS'] = False
 
-app.config['DEBUG_TB_INTERCEPT_REDIRECTS'] = True
+SESSION_RES = 'response'
 
-# test comment
 @app.route('/')
 def index():
     """Show homepage"""
+    return render_template("surveys.html")
+
+@app.route('/start', methods=["POST"])
+def setup_session():
+    """Sets up a session if one is not already made"""
+    if session.get(SESSION_RES) == None:
+        session[SESSION_RES] = []
```

Comment:
`something == None` is not pythonic. You would say `something is not None`. Read more [here].(http://jaredgrubb.blogspot.com/2009/04/python-is-none-vs-none.html)
```suggestion
    if not session.get(SESSION_RES):
        session[SESSION_RES] = []
```